felix_requirements.txt

```diff
@@ -1,4 +1,4 @@
 gevent
 greenlet
 netaddr
-python-etcd==0.3.3-calico-1
+python-etcd==0.3.3-calico-2
```

Comment:
Also, if we're not pointing at the github repository, we should be using >=, not ==.